packages/network-query/queries/network/deployments.gql

```diff
@@ -35,12 +35,36 @@ query GetDeployment($deploymentId: String!) {
   }
 }
 
-query GetDeploymentIndexers($offset: Int, $deploymentId: String!) {
+query GetDeploymentIndexers(
+  $first: Int = 20
+  $offset: Int
+  $deploymentId: String!
+  $orderby: [DeploymentIndexersOrderBy!] = LAST_EVENT_ASC
+  $search: Boolean! = false
+  $indexerId: String! = ""
+) {
   deploymentIndexers(
-    first: 20
+    first: $first
     offset: $offset
+    orderBy: $orderby
     filter: { deploymentId: { equalTo: $deploymentId }, status: { notEqualTo: TERMINATED } }
-  ) {
+  ) @skip(if: $search) {
+    totalCount
+    nodes {
+      ...DeploymentIndexerFields
+    }
+  }
+
+  deploymentIndexersBySearch: deploymentIndexers(
+    first: $first
+    offset: $offset
+    orderBy: $orderby
+    filter: {
+      deploymentId: { equalTo: $deploymentId }
+      status: { notEqualTo: TERMINATED }
+      indexerId: { equalTo: $indexerId }
```

Comment:
Feels it doesn't worth the effort, can just separate them into two queries

Reply:
@ianhe8x ok